masonry_winit/src/app_driver.rs

```diff
@@ -98,35 +97,23 @@ impl DriverCtx<'_, '_> {
     ///
     /// # Panics
     ///
-    /// Panics if the window cannot be found or the window is not in the rendering state.
+    /// Panics if the window cannot be found.
     pub fn window_handle(&self, window_id: WindowId) -> &WindowHandle {
         let window = self.state.window(window_id);
-        let WindowState::Rendering { handle: window, .. } = &window.state else {
-            panic!(
-                "window with id {window_id:?} is in {:?} state, expected Rendering state",
-                window.state
-            );
-        };
-        window
+        &window.handle
```

Comment:
I am not sure if this is the right choice, however, if we still want to panic if it's in suspended state, then we can add an if check here.

Reply:
This seems reasonable enough, I think.